defensics/tcp_data_handler.py

```diff
@@ -0,0 +1,37 @@
+import threading
+import logging
+
+from defensics.coap_proxy import CoapProxy
+from defensics.tcp_server import TCPServer
+
+class CoapAutomationHandler(threading.Thread):
```

Comment:
Why have CoapAutomationHandler and data_handler?

Reply:
Oh, sure, missed leftover